laterpay/asset_sources/js/laterpay-post-view.js

```diff
@@ -173,7 +173,6 @@
                 // select radio input by clicking on a container
                 $o.body
                     .on('click', $o.optionContainer, function (e) {
```

Comment:
```suggestion
                    .on('click', $o.optionContainer, function () {
```
We can remove `e` as it's not needed anymore.